src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -4,11 +4,15 @@ import * as Lint from 'tslint';
 class TrailingCommaInterfaceWalker extends Lint.RuleWalker {
    private static FAILURE_STRING = 'Trailing comma is required in interfaces properties.';
    public visitInterfaceDeclaration(node: ts.InterfaceDeclaration) {
+    const potentialDelimiters = [';'];
     node.members.map((propertySignature) => {
-        if (propertySignature.getText().slice(-1) !== ',') {
-            this.addFailure(this.createFailure(propertySignature.getStart(),
-                                               propertySignature.getWidth(),
-                                               TrailingCommaInterfaceWalker.FAILURE_STRING));
+        const lastChar = propertySignature.getText().slice(-1);
+        if (lastChar !== ',') {
+          const lastCharPosition = propertySignature.getStart() + propertySignature.getWidth() - 1;
+          const replacementText = (potentialDelimiters.indexOf(lastChar) === -1) ? `${lastChar},` : ',';
```

Comment:
`potentialDelimiters` feels like an over-abstraction; it'd take a language change to add anything to this array!

Use `arr.includes(foo)` if you're not going to use the index.

Favour truthy comparisons.  Most conditions are truth-first, so it's easy to miss inversions.  eslint has a [no-negated-condition](http://eslint.org/docs/rules/no-negated-condition) rule to enforce this (I wonder if they gave it an intentionally bad name?)

So, something like:
```
const replacementText = (lastChar === ';' ? ',' : `${lastChar},`;
```